hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSetProcessor.java

```diff
@@ -353,20 +357,43 @@ public StructType getExternalFileSchema(String storedAs, String location){
 
             }
             if (storedAs.toLowerCase().equals("t")) {
-                schema =  SpliceSpark.getSession().read().csv(location).schema();
+                 schema = getCsvSchema(location);
             }
         }
 
         return schema;
     }
 
+
+    private StructType getCsvSchema(String location) throws IOException {
+        InputStream is = null;
```

Comment:
Are we assuming all csv files will have a "schema" file? What happens when they don't?
Couldn't we just disable checking the types when it's a CSV file?